app/stacks/MasterDetailStack/ModalContainer.tsx

```diff
@@ -25,8 +28,21 @@ const styles = StyleSheet.create({
 });
 
 export const ModalContainer = ({ navigation, children, theme }: IModalContainer): JSX.Element => {
+	const keyboardHeight = useKeyboardHeight();
 	const { height } = useWindowDimensions();
+	const { isLandscape } = useOrientation();
 	const modalHeight = sharedStyles.modalFormSheet.height;
+
+	let heightModal: number;
+
+	if (modalHeight > height) {
+		heightModal = height;
+	} else if (isLandscape && isAndroid && keyboardHeight > 0) {
+		heightModal = height - keyboardHeight - 32; // 32 is to force a padding
```

Comment:
Instead of a 32 with a comment, create a const with a meaningful name.

```tsx
// at the top of the file
const MODAL_MARGIN = 32;

...

heightModal = height - keyboardHeight - MODAL_MARGIN
```